string namespace
string type
geometry_msgs/PoseStamped robot_pose